repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -91,7 +91,14 @@ class ExecutorClassLoader(conf: SparkConf, classUri: String, parent: ClassLoader
       inputStream.close()
       Some(defineClass(name, bytes, 0, bytes.length))
     } catch {
-      case e: Exception => None
+      case e: FileNotFoundException =>
+        // We did not find the class
+        logDebug(s"Did not load class $name from REPL class server at $uri", e)
```

Comment:
It is actually a remote URI: Have a look at the [usage site][1]. It can either point to an HTTP server or an HDFS.

[1]: https://github.com/gzm0/spark/blob/4fa0582e90ebcf3cbc19971e26c8cda83d5613e1/repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala#L47